pkgs/top-level/python-packages.nix

```diff
@@ -15392,7 +15392,7 @@ self: super: with self; {
   tensorflow-bin = callPackage ../development/python-modules/tensorflow/bin.nix {
     inherit (pkgs.config) cudaSupport;
     # https://www.tensorflow.org/install/source#gpu
-    cudaPackages = pkgs.cudaPackages_11;
+    cudaPackages = pkgs.cudaPackages_12;
```

Comment:
can we drop this as `cudaPackages` is `cudaPackges_12` ?

Reply:
Technically we could, but as this package is precompiled and hardcoded to use a specific CUDA version (e.g. `libcudart.so.12`), I'm not sure what the benefit would be.

The only time the CUDA version can be changed without breaking the package is when the package is updated, and when that happens it can be bumped like this.

Reply:
Yeah I guess we should ~~get rid of all -bin packages already~~ pin -bin packages to concrete cuda releases